src/Services/Indexer/IndexBuilderService.php

```diff
@@ -28,7 +26,7 @@ public function __construct(string $serverType, $debug = false)
 
         $locale = env('SB_LOCALES_DEFAULT', 'nl');
```

Comment:
This is the BACKEND default locale. Not the site default locale,

Reply:
As $this->defaultLocale was not used I removed both lines.